### PR TITLE
fix the nunjucks issue in windows platform.

### PR DESCRIPTION
**Pull request by hwanseung**
resolve the nunjucks issue in windows platform.
and generated header and cpp files will be located ```bacardi/build/Release/gen/examples```.

ISSUE=#26

### CODE REVIEW

File: generator/main.ts [hunk 1]
```diff
@@ -15,19 +15,29 @@
  */
 
 import * as file from './base/file';
+import * as fs from 'fs';
```
Comment (romandev) [inline generator/main.ts]: Please remove this. it looks unused. :)

File: generator/main.ts [hunk 2]
```diff
@@ -15,19 +15,29 @@
...
 async function main(idl_files: Array<string>) {
   for (let idl_file of idl_files) {
     let parsedData = webidl.parse(await file.read(idl_file));
     let idl_interface: idls.Interface = new idls.InterfaceImpl(parsedData[0]);
     nunjucks.configure({ trimBlocks:true, lstripBlocks: true });
-    await file.write('build/calculator_bridge.h',
-        nunjucks.render('./template/interface_header.njk', idl_interface));
-    await file.write('build/calculator_bridge.cc',
-        nunjucks.render('./template/interface_cpp.njk', idl_interface));
+    let template_folder = __dirname + './../../../template/';
```
Comment (romandev) [inline generator/main.ts]: Please use const instead of let.

File: generator/main.ts [hunk 3]
```diff
@@ -15,19 +15,29 @@
...
+    let gen_folder = path.resolve(__dirname, '../gen/examples');
```
Comment (romandev) [inline generator/main.ts]: ditto

File: generator/main.ts (hunk 1 above)
Comment (hwanseung) [reply generator/main.ts]: done.

File: generator/main.ts (hunk 2 above)
Comment (hwanseung) [reply generator/main.ts]: done.

File: generator/main.ts (hunk 3 above)
Comment (hwanseung) [reply generator/main.ts]: done.

**Review by romandev — APPROVED** [review]
lgtm

**Pull request merged**